cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -20,7 +20,8 @@ func getStateCommitment(commits storage.Commits, blockHash flow.Identifier) (flo
 
 func extractExecutionState(dir string, targetHash flow.StateCommitment, outputDir string, log zerolog.Logger) error {
 
-	led, err := complete.NewLedger(dir, 1000, &metrics.NoopCollector{}, log, nil, complete.DefaultPathFinderVersion)
+
+	led, err := complete.NewLedger(dir, 1000, &metrics.NoopCollector{}, log, nil, 0)
```

Comment:
This loads ledger with path finder version zero and export as ledger with path finder version 1